code-challenges.test.js

```diff
@@ -11,28 +11,113 @@
 // Run the file with the following command: $ yarn jest
 
 
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// --------------------1) Create a function and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
+// TDD
+// RED
+// 1- Write the test
+// 2- Run a test "yarn jest" to see failed test. 
 
-const fibLength1 = 6
+describe("number", () => {
+    it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+      expect(number()).toEqual("numbers of the Fibonacci sequence")
+    })  
+})
+
+//  FAIL  jest-dawit/jest.test.js
+  // ● number › returns an array that length containing the numbers of the Fibonacci sequence
+
+  // ReferenceError: number is not defined
+
+// GREEN
+// 3- Write the code 
+
+//Psudocode:
+// Write a function called fibLength1
+// Parameter is Fibonacci numbers of 6.
+// Expected out put of an array containg the following numbers [1, 1, 2, 3, 5, 8]
+
+//const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
```

Comment:
Look over the structure of functions. Use a parameter vice the argument. The parameter will allow the function to be reusable.